src/components/compound/DlCharts/charts/DlConfusionMatrix/types.ts

```diff
@@ -0,0 +1,18 @@
+export interface MatrixCell {
```

Comment:
These types will be exported. please add a `Dl` prefix to them. and make them unique